f3/granite.go

```diff
@@ -728,6 +1039,99 @@ func (c *convergeState) findMinTicketProposal() ECChain {
 	return minValue
 }
 
+// BitSet is a bitset implementation.
+type BitSet struct {
```

Comment:
Please import and use https://github.com/filecoin-project/go-bitfield/blob/master/bitfield.go instead

Reply:
done.